packages/svelte/tests/runtime-runes/samples/each-was-empty/_config.js

```diff
@@ -0,0 +1,38 @@
+import { flushSync } from 'svelte';
+import { test } from '../../test';
+
+// https://github.com/sveltejs/svelte/issues/13550
+// https://github.com/sveltejs/svelte/pull/13553
+export default test({
+	html: `<button>clicks: 0</button><button>undefined</button><button>null</button><button>empty</button><button>[1,2,3]</button><ul><li>count = <span>0</span></li></ul>`,
+
+	async test({ assert, target }) {
+		const [increment, set_undefined, set_null, set_empty, set_list] =
+			target.querySelectorAll('button');
+
+		const [span] = target.querySelectorAll('span');
+
+		let count = 0;
+		assert.equal(span.innerHTML, `${count}`);
+
+		for (const button of [set_undefined, set_null, set_empty, set_undefined, set_null, set_empty]) {
+			flushSync(() => {
+				button.click();
+			});
+			flushSync(() => {
+				increment.click();
+				count++;
+			});
+			assert.equal(span.innerHTML, `${count}`);
+		}
+
+		flushSync(() => {
+			set_list.click();
+		});
+
+		assert.htmlEqual(
+			target.innerHTML,
+			`<button>clicks: ${count}</button><button>undefined</button><button>null</button><button>empty</button><button>[1,2,3]</button><ul><li>item : 1</li><li>item : 2</li><li>item : 3</li></ul>`
+		);
```

Comment:
I know it's a bit more of manual work but i think tests should be as manual as possible...if for some reason in the future we break the logic of the test it could fail even if we don't have a problem in the library. Can you please rewrite this test with all manual actions?

Reply:
I just modified it